### ISSUE TITLE
Icinga Web 2 Role - Condition check fails in old versions

### ISSUE THREAD

**Issue by tbauriedel** [Icinga/ansible-collection-icinga#218]
The condition [here](https://github.com/Icinga/ansible-collection-icinga/blob/main/roles/icingaweb2/tasks/main.yml#L19) fails in old versions.

```
TASK [icinga.icinga.icingaweb2 : Gather module packages] ********************************************************************************************************************************************************
fatal: [*****]: FAILED! => {"msg": "The conditional check 'icingaweb2_modules is defined and icingaweb2_module_packages[item.key] is defined and item.value.enabled is true and item.value.source == \"package\"' failed. The error was: template error while templating string: no test named 'true'. String: {% if icingaweb2_modules is defined and icingaweb2_module_packages[item.key] is defined and item.value.enabled is true and item.value.source == \"package\" %} True {% else %} False {% endif %}\n\nThe error appears to be in '/home/******/.ansible/collections/ansible_collections/icinga/icinga/roles/icingaweb2/tasks/main.yml': line 15, column 3, but may\nbe elsewhere in the file depending on the exact syntax problem.\n\nThe offending line appears to be:\n\n\n- name: Gather module packages\n  ^ here\n"}
```

For backwards compatibility it would be better to change that to something like this.
```
item.value.enabled | bool == true
```

Ubuntu 20.04

$ pip list
ansible                6.7.0
ansible-core           2.13.13
Jinja2                 3.1.2

**Comment by mkayontour** [comment]
Thanks for pointing it out. I'll change it.

**Comment by mkayontour** [comment]
Fixed with this commit 9397d18, thanks again for the issue.
